MediaContent/Observer/CatalogProductSaveAfter.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaContent\Observer;
+
+use Magento\Catalog\Api\Data\ProductInterface;
+use Magento\Framework\Event\Observer;
+use Magento\Framework\Event\ObserverInterface;
+use Magento\Framework\Exception\IntegrationException;
+use Magento\MediaContent\Model\ContentProcessor;
+
+/**
+ * Observe the catalog_product_save_after event and run processing relation between media content and media asset
+ */
+class CatalogProductSaveAfter implements ObserverInterface
```

Comment:
Please avoid using the event name in the name of the observer class name, it can potentially be used to observe different event
```suggestion
class CatalogProduct implements ObserverInterface
```